app/ast1060-starter/src/main.rs

```diff
@@ -33,13 +44,10 @@ fn main() -> ! {
             .enbl_armtdofn_pin().bit(true)
     });
 
```

Comment:
Configure the non-cache area for ram_nc region.

```suggestion
    peripherals.scu.scua58().modify(|_, w| {
        // Disable Cache
        w.cache_enbl().bit(false)
    });
    peripherals.scu.scua50().write(|w| unsafe {
        w.bits(0x000f_ffff)
    });

    peripherals.scu.scua54().write(|w|  unsafe {
        w.bits(0x8660_0000)
    });
    peripherals.scu.scua58().modify(|_, w| {
        // Enable Cache
        w.cache_enbl().bit(true)
    });
```